src/Policy/LimitPolicyInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\RateLimiter\Policy;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+interface LimitPolicyInterface
+{
+    public function fingerprint(ServerRequestInterface $request): string;
```

Comment:
I wonder if naming it as counterID or something related to where it's used will be better to understand what is it for or not.